AdyenActions/Actions/QRCodeAction.swift

```diff
@@ -19,7 +19,7 @@ public enum QRCodePaymentMethod: String, Codable, CaseIterable {
 public struct QRCodeAction: PaymentDataAware, Decodable {
     
     /// The `paymentMethodType` for which the QR code is presented.
-    public let paymentMethodType: QRCodePaymentMethod
+    public private(set) var paymentMethodType: QRCodePaymentMethod = .pix
```

Comment:
this shouldn't be always `.pix`, its a generic `QRCodeAction` as we discussed :-)

Reply:
Right. Let me change this :)